app-crypt/argon2/files/argon2-20160821-makefile-install-target.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/Makefile b/Makefile
+index e9d27c8..90c1818 100644
+--- a/Makefile
++++ b/Makefile
+@@ -80,11 +80,25 @@ endif
+ 
+ LIB_SH := lib$(LIB_NAME).$(LIB_EXT)
+ LIB_ST := lib$(LIB_NAME).a
++LIBRARIES = $(LIB_SH) $(LIB_ST)
++HEADERS = include/argon2.h
+ 
+-.PHONY: clean dist format $(GENKAT)
```

Comment:
It's moved to a different location (this patch is generated from a git commit that adds the install target in upstream in a newer version)